pixeltable/exprs/expr.py

```diff
@@ -681,6 +688,48 @@ def __rmod__(self, other: object) -> 'exprs.ArithmeticExpr':
     def __rfloordiv__(self, other: object) -> 'exprs.ArithmeticExpr':
         return self._rmake_arithmetic_expr(ArithmeticOperator.FLOORDIV, other)
 
+    @classmethod
+    def is_str(cls, object: Any) -> bool:
```

Comment:
not sure where is the best place to put this or there is an existing method 
I want to see if given object is int/str or expression that returns int/str